docs/config/docSites.json

```diff
@@ -89,7 +89,7 @@
       "presentation-frontend": "presentation-frontend",
       "quantity": "core-quantity",
       "reality-data-client": "reality-data-client",
-      "transformer": "core-transformer",
+      "transformer": "transformer",
```

Comment:
imodel-transformer?

Reply:
I will generate a new doc build to test

Reply:
actually can we keep the doc artifact name as just `transformer`, I suppose it doesn't matter because of the alias here, and we can always change it in the future. Thoughts?

Reply:
Sounds better to me.

Reply:
the key can stay transformer, but the value should be the pkg name, imodel-transformer